index.js

```diff
@@ -386,7 +386,7 @@
         return x != null && (
           R.type(type.prototype['@@type']) === 'String' ?
             x['@@type'] === type.prototype['@@type'] :
-            R.type(x) === R.nth(1, R.match(/^function (\w*)/, String(type)))
+            R.type(x) === R.nth(1, R.match(/function (\w*)/, String(type)))
```

Comment:
I thought about such pattern but insertion of "\s*" still does not make it safe. Because according to specification there can be any combination of spaces, newlines and semicolons. So it should be something like this: `/^[;\s]*function\s*(\w*)/`